### PR TITLE
Added port subcommand.

### PR DESCRIPTION
**Pull request by eyzeedev**
Doesn't deal with port ranges, and needs some work, but really just opening up a dialogue to drive through the rest of the implementation.

### CODE REVIEW

**Comment by muayyad-alsadi** [comment]
It seems that you undo some other work. You would you update your branch from latest devel

File: podman_compose.py [hunk 1]
```diff
@@ -904,9 +905,6 @@ def container_to_args(compose, cnt, detached=True):
         if is_str(entrypoint):
             entrypoint = shlex.split(entrypoint)
         podman_args.extend(["--entrypoint", json.dumps(entrypoint)])
-    platform = cnt.get("platform", None)
```
Comment (muayyad-alsadi) [inline podman_compose.py:907]: Why remove this?

**Comment by eyzeedev** [comment]
Rebased onto upstream devel with no changes. Reinstated accidental deletion.

**Comment by eyzeedev** [comment]
It remains to implement port-ranges and indexes.

**Pull request merged**